apps/chat/src/view/chatView.scss

```diff
@@ -39,6 +39,11 @@
 .chatMessage {
   padding: $sizeS $sizeM;
   border-radius: $sizeM;
+
+  .vuiSpinner {
+    display: inline-block;
+    margin-left: 8px;
+  }
```

Comment:
Per my above comment I think we can remove this. In latest version of VUI, VuiSpinner is now inline-block already.